### PR TITLE
Ban feature state handling

### PR DESCRIPTION
**Pull request by rottabonus**
This PR has actions implemented in workshop for the ban feature.

### CODE REVIEW

File: src/Screens/Main/Chat/Title.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ type Props = OwnProps & DispatchProps & StateProps;
 
 const Title: React.FC<Props> = ({ style, onPress, name, buddyId }) => {
   const color = getBuddyColor(buddyId);
+
   return (
```
Comment (pheis) [inline src/Screens/Main/Chat/Title.tsx:28]: Nice to have the 'zen' line : D

**Review by pheis — APPROVED** [review]
Good job :)!

File: src/Screens/Main/Chat/Title.tsx (hunk 1 above)
Comment (pheis) [reply src/Screens/Main/Chat/Title.tsx:28]: But not really part of this PR, but doesn't matter.

Comment (rottabonus) [reply src/Screens/Main/Chat/Title.tsx:28]: Yep, decided to let it be =)

**Pull request merged**